include/deal.II/base/data_out_base.h

```diff
@@ -947,13 +947,29 @@ namespace DataOutBase
     bool print_date_and_time;
 
     /**
-     * A data type providing the different possible zlib compression levels.
+     * A data type providing the different possible zlib compression
+     * levels. These map directly to constants defined by zlib.
      */
     enum ZlibCompressionLevel
     {
+      /**
+       * Do not use any compression.
+       */
       no_compression,
+      /**
+       * Use the fastest available compression algorithm. This is usually the
+       * right flag to use.
```

Comment:
That's a reasonable point: I don't have anything scientific to back this claim up.